website/routes.py

```diff
@@ -197,6 +198,13 @@ def make_url_map(app):
             OsfWebRenderer('public/pages/meeting_landing.mako'),
         ),
 
+        Rule(
+            '/api/v1/citation_styles/',
+            'get',
+            citation_views.styles,
```

Comment:
Given that `url_for` will use the function name for reverse lookup, we might want to give this a more descriptive name. Perhaps `citation_styles`?